device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D112C8/pg_profile_lookup.ini

```diff
@@ -17,22 +17,22 @@
 #  PG lossless profiles.
 # speed cable  size   xon    xoff   threshold
   10000    5m 19456  19456  24576          0
-  25000    5m 19456  19456  24576          0
```

Comment:
@stephenxs, the 2km support is only for SN4600C-C64? If that is the case, let's create a separate pg_profile_lookup.ini for all other 4600skus without the 2km and share them. right now this file is shared by all 4600C skus

Reply:
Yes. We had an internal discussion with Arch and agreed on using the updated algorithm to calculate the headroom size for all 4600c SKUs. So this common pg profile lookup.ini is updated
Will that do?